bin/run_tests.sh

```diff
@@ -4,6 +4,13 @@ set -e
 base=$(pwd)
 failed=0
 
+# run snyk tests locally. Skip in bamboo due to authentication token requirement.
+echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
+snyk iac test || true
```

Comment:
Currently the tests will continue even if there are errors raised in snyk because the issues can be critical and not straightforward which might require creating additional cards for implementation as seen in snyk issues in the past.

Reply:
Because I'm me, I lean towards loud errors. We currently hold up PRs for snyk errors, so it feels consistent to raise an error in scripts.

Reply:
Per conversation, need to discuss with Jon V. on how often we can run IAC tests based on their license.